source/core/types/ut_reporter_base.tps

```diff
@@ -5,6 +5,21 @@ create or replace type ut_reporter_base force authid current_user as object
   constructor function ut_reporter_base(self in out nocopy ut_reporter_base, a_output ut_output default ut_output_dbms_output()) return self as result,
 
   member procedure print_text(self in out nocopy ut_reporter_base, a_text varchar2),
+
```

Comment:
Ok or simply put it into `ut_documentation_reporter`. If you feel it's going to be reused, abstraction is ok for me as well.